operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1596,6 +1596,73 @@ memcachedExporter:
       drop: [ALL]
     allowPrivilegeEscalation: false
 
+admin-cache:
+  # -- Specifies whether memcached based admin-cache should be enabled
+  enabled: false
+
+  # -- Total number of admin-cache replicas
+  replicas: 1
+
+  # -- Port of the admin-cache service
+  port: 11211
+
+  # -- Amount of memory allocated to admin-cache for object storage (in MB).
+  allocatedMemory: 64
```

Comment:
Based on [this](https://docs.google.com/document/d/11Nv9FYLld_VKSWUjsjuomTCdxut3e28vPYuQT6k3ipg/edit#bookmark=id.rvgn03j9pl3k), probably we can set it even lower?

Reply:
There's a rough estimation that total admin bucket size is <2Mb. 

This is however based on our `ge-metrics` dev environment and may not be aligned with production datasets. 

So for me, it's ok to leave it as it is. 👍